gruntfile.js

```diff
@@ -201,7 +201,8 @@ module.exports = function (grunt) {
     protractor: {
       options: {
         configFile: 'protractor.conf.js',
-        keepAlive: true,
+        // Stops Grunt process if a test fails
+        keepAlive: false,
```

Comment:
The default is false, right? I'd rather just remove it because it's the behavior we expect and Grunt isn't as clear as it could be anyway.